bundles/org.eclipse.passage.lic.features/META-INF/MANIFEST.MF

```diff
@@ -2,7 +2,7 @@ Manifest-Version: 1.0
 Automatic-Module-Name: org.eclipse.passage.lic.features
 Bundle-ManifestVersion: 2
 Bundle-SymbolicName: org.eclipse.passage.lic.features
-Bundle-Version: 1.0.2000.qualifier
+Bundle-Version: 1.0.200.qualifier
```

Comment:
nope, we cannot decrease, 
but I can leave it here since we will have 2.0.0 soon for this feature

Reply:
Done